pynemo/pynemo_exe.py

```diff
@@ -42,7 +42,7 @@ def main():
     #Logger
     #logger = logging.getLogger(__name__)
     t0 = time.time()
-    cProfile.runctx("f(x, y)",{'f': profile.process_bdy, 'x': setup_file, 'y': mask_gui}, {}, 'pynemo_stats')
+    cProfile.runctx("f(x, y)",{'f': profiler.process_bdy, 'x': setup_file, 'y': mask_gui}, {}, 'pynemo_stats')

```

Comment:
This is a name change. `cProfile` has a method called `profile` so duplication is bad.